packages/app-store/cal-ai/api/_getAdd.ts

```diff
@@ -33,6 +33,21 @@ export async function getHandler(req: NextApiRequest, res: NextApiResponse) {
     },
   });
 
+  // BAD TEMP HACK
+  process.env.CALAI_URL = "http://localhost:3005";
+  process.env.CALAI_API_KEY = "12345";
```

Comment:
Couldn't figure out how to get the App Store env working... This is currently blocking!